server/game/cards/characters/04/aerondamphair.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../../drawcard.js');
+
+class AeronDamphair extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCharacterSaved: () => true
```

Comment:
Per my event rename suggestion, you should check that the card being saved is a character.